cranelift/src/clif-util.rs

```diff
@@ -62,6 +62,26 @@ fn add_verbose_flag<'a>() -> clap::Arg<'a, 'a> {
     Arg::with_name("verbose").short("v").help("Be more verbose")
 }
 
+arg_enum! {
+    #[derive(Clone, Copy, Debug, PartialEq)]
+    pub enum UseTerminalColor {
+        Yes,
+        No,
+        Always
```

Comment:
I just checked with `git` and their options are `always|never|auto`. I think it is worth being consistent with that, since people are more likely used to their options than something we make up ourselves. sorry for getting it wrong initially!